src/node_quic_crypto.h

```diff
@@ -260,62 +97,11 @@ int ALPN_Select_Proto_CB(
     unsigned int inlen,
     void* arg);
 
-int Client_Transport_Params_Add_CB(
-    SSL* ssl,
-    unsigned int ext_type,
-    unsigned int context,
-    const unsigned char** out,
-    size_t* outlen,
-    X509* x,
-    size_t chainidx,
-    int* al,
-    void* add_arg);
-
 int TLS_Status_Callback(SSL* ssl, void* arg);
 
-int Server_Transport_Params_Add_CB(
-    SSL* ssl,
-    unsigned int ext_type,
-    unsigned int context,
-    const unsigned char** out,
-    size_t* outlen,
-    X509* x,
-    size_t chainidx,
-    int* al,
-    void* add_arg);
-
-void Transport_Params_Free_CB(
-    SSL* ssl,
-    unsigned int ext_type,
-    unsigned int context,
-    const unsigned char* out,
-    void* add_arg);
-
-int Client_Transport_Params_Parse_CB(
-    SSL* ssl,
-    unsigned int ext_type,
-    unsigned int context,
-    const unsigned char* in,
-    size_t inlen,
-    X509* x,
-    size_t chainidx,
-    int* al,
-    void* parse_arg);
-
-int Server_Transport_Params_Parse_CB(
-    SSL* ssl,
-    unsigned int ext_type,
-    unsigned int context,
-    const unsigned char* in,
-    size_t inlen,
-    X509* x,
-    size_t chainidx,
-    int* al,
-    void* parse_arg);
-
 bool GenerateRetryToken(
     uint8_t* token,
-    size_t* tokenlen,
+    size_t& tokenlen,
```

Comment:
Does the linter allow this? We generally have a policy against non-const references, and I think this is a case where using a pointer made sense